tools/parse-glibc.py

```diff
@@ -19,9 +19,9 @@ def checknonnull(cfg, functionName, nonnull):
             continue
         argpos2 = functionCfg.find('</arg>', argpos1)
         notnullpos = functionCfg.find('not-null', argpos1)
-        if notnullpos > 0 and notnullpos < argpos2:
+        if 0 <= notnullpos < argpos2:

```

Comment:
ok, it's better it is more pythonic (however as a c/c++ programmer it's easy to think it means something else).